web/client/src/App.css

```diff
@@ -31,7 +33,7 @@ label {
   flex-direction: row;
   justify-content: flex-end;
   text-align: right;
-  width: 50px;
+  width: 70px;
```

Comment:
We should try and use rem or % values here to support different screen sizes

Reply:
done